src/new-client/src/components/DraggableComponent/DraggableComponent.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from "styled-components"
+import { RefObject, useContext, useState } from "react"
+import { TearOutContext } from "@/App/TearOutSection/tearOutContext"
+import { tearOutSection, Section } from "@/App/TearOutSection/state"
+import { canDrag } from "@/components/DraggableComponent/canDrag"
+import { tearOut } from "@/App/LiveRates/Tile/TearOut/state"
+
+interface DraggableComponentProps {
+  children: JSX.Element[] | JSX.Element
+  section: Section
+  isTile?: boolean
+  ref?: RefObject<HTMLDivElement>
+  isTornOut?: boolean
+  symbol?: string
+}
+
+export const DraggableComponent: React.FC<DraggableComponentProps> = (
+  props: DraggableComponentProps,
+) => {
+  //stopPropagation() is not enough to avoid the parent (in case we are dragging a Tile) also tearing out
+  //this state makes sure that the drag event started and finished succesfully
+  const [finishedDrag, setFinishedDrag] = useState(false)
+
+  const tearOutContext = useContext(TearOutContext)
+  const symbol = props.symbol ? props.symbol : ""
+  const ref = props.ref ? props.ref : { current: null }
+
+  const eventHandler = (event: React.DragEvent) => {
+    props.isTile
+      ? tearOut(symbol, !props.isTornOut, ref.current!)
+      : tearOutSection(!tearOutContext.isTornOut, props.section)
```

Comment:
As above, I think this could just be `tearOutSection(true, props.section)` without the need for dragging back in (same goes for `tearOut` above)

Reply:
Fixed